apps/base/utils.py

```diff
@@ -32,3 +33,11 @@ def __call__(self, instance, filename):
         filename = '{}{}'.format(uuid.uuid4(), extension)
         filename = os.path.join(self.path, filename)
         return filename
+
+
+def get_model_object(model_name, pk):
```

Comment:
I think this is not what I meant. I think what I meant was a function which wraps this piece of code for a model and then returns that function